docs/5_2_scheduled_execution_api.adoc

```diff
@@ -68,22 +68,26 @@ This function can be called before scheduling a model partition for variables as
 Function <<get-and-set-variable-values,`fmi3Get{VariableTypeExclClock}`>>::
 These functions can be called after the computation of a model partition for variables assigned to that model partition via its associated <<Clock>> and all variables not associated to a <<Clock>> (global variables).
 
-+
-Set/get operations must be atomic for a single variable.
-
-+
 _[Because of real-time constraints, the computational effort has to be predictable for all operations in Scheduled Execution._
 _Therefore, all computationally expensive operations to compute a model partition have to be contained within the <<fmi3ActivateModelPartition>> function._
 _The simulation algorithm can assume that <<get-and-set-variable-values,`fmi3Get{VariableTypeExclClock}`>> and <<get-and-set-variable-values,`fmi3Set{VariableTypeExclClock}`>> operations are not computationally expensive._
 _It is recommended, to call <<get-and-set-variable-values,`fmi3Set{VariableTypeExclClock}`>> and <<get-and-set-variable-values,`fmi3Get{VariableTypeExclClock}`>> in the same task as <<fmi3ActivateModelPartition>>.]_
 
-+
 The restrictions related to variable <<causality>> and <<variability>> defined for <<StepMode>> in <<fmi-for-co-simulation,`Co-Simulation`>> apply.
 
-+
 It is not allowed to call <<get-and-set-variable-values,`fmi3Get{VariableTypeExclClock}`>> functions after <<get-and-set-variable-values,`fmi3Set{VariableTypeExclClock}`>> functions without an <<fmi3ActivateModelPartition>> call in between.
 _[The reason is to avoid different interpretations of the caching, since contrary to <<fmi-for-model-exchange, `FMI for Model Exchange`>>, <<fmi3ActivateModelPartition>> will perform the actual calculation instead of <<get-and-set-variable-values,`fmi3Get{VariableTypeExclClock}`>>, and therefore, dummy algebraic loops at communication points cannot be handled by an appropriate sequence of <<get-and-set-variable-values,`fmi3Get{VariableTypeExclClock}`>> and <<get-and-set-variable-values,`fmi3Set{VariableTypeExclClock}`>> calls as for Model Exchange.]_
 
+To ensure predictable behavior every variable should be assigned uniquely to a model partition via its associated <<Clock>>.
+Observe two examples:
```

Comment:
Shouldn't we format the examples as non-normative? 
Just as _[Examples: ...

Reply:
Also, the previous mention of the unique assignment was made as a strong recommendation.
So the text "To ensure predictable..." and subsequent examples should be non-normative, I think.

Reply:
@andreas-junghanns would you please decide on the formating?

Reply:
.... here a various comments below.

Reply:
I put this in _[ ... ]_